backend/globaleaks/jobs/delivery.py

```diff
@@ -91,21 +94,35 @@ def write_plaintext_file(sf, dest_path):
         log.err("Unable to create plaintext file %s: %s", dest_path, excep)
 
 
-def write_encrypted_file(key, sf, dest_path):
+def write_encrypted_file(session, key, sf, dest_path):
+    antivirus_clamd_ip = ConfigFactory(session, 1).get_val('antivirus_clamd_ip')
+    antivirus_clamd_port = ConfigFactory(session, 1).get_val('antivirus_clamd_port')
+    antivirus_enabled = ConfigFactory(session, 1).get_val('antivirus_enabled')
+    af = FileAnalysis(host = antivirus_clamd_ip, port = antivirus_clamd_port)
+    status_file = EnumStateFile.verified
     try:
         with sf.open('rb') as encrypted_file, \
              GCE.streaming_encryption_open('ENCRYPT', key, dest_path) as seo:
+            id_file = dest_path.split('/')[-1]
             chunk = encrypted_file.read(abstract.FileDescriptor.bufferSize)
```

Comment:
@alessiofranceschini : still this appear to read a file of 1GB chunk by chunk and analyze a single chunk. This will result in failure in recognizing viruses.